cluster/Vagrantfile

```diff
@@ -102,7 +102,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       elsif node_os == UBUNTU
         c.vm.box = 'ubuntu/xenial64'
       else
-        c.vm.box = 'centos/7'
+        if ENV['CONTIV_KUBEADM']
+           c.vm.box = 'contiv/kubeadm-centos'
```

Comment:
@rchirakk, how much time does this save? What is the process to update the kubeadm version or k8s version?

Right now the k8s version is configurable and updating it is very simple. Most users using kubeadm also go the the new versions very soon.